odl/diagnostics/operator.py

```diff
@@ -75,29 +81,29 @@ def norm(self):
         Wikipedia article on `Operator norm
         <https://en.wikipedia.org/wiki/Operator_norm>`_.
         """
-        print('\n== Calculating operator norm ==\n')
+        if self.verbose:
```

Comment:
This repetative "if self.verbose: print(something)" could be changed to adding a function 

```
def log(self, string):
    if self.verbose:
        print(string)
```

that way we can remove quite some lines of code and also make this extendable in the future.